ui/app/src/components/Footer.tsx

```diff
@@ -35,7 +35,7 @@ export default function Footer(): JSX.Element {
             <Github sx={{ verticalAlign: 'bottom' }} />
           </a>
         </li>
-        <li>
+        <li data-happo-hide>
```

Comment:
Hiding the version number in the footer from happo because it is something that will change somewhat regularly, leading to flaky snapshots.

Reply:
Nice, TIL how to hide elements that could break our visual regression tests